### DISCUSSION TITLE
User Invite Email Failed - Can't re-invite for 7 days

### DISCUSSION THREAD

**Discussion by Shadowfita** [Budibase/budibase#12574] [Help]
Hi,

I have a user who did not receive the invite email (although 4 others did) and it seems there is no way to cancel this invite, or re-submit an invite, until the 7-day grace-period finishes and the invite expires.

Is there any workaround available?

**Comment by ConorWebb96** [comment]
Hey @Shadowfita,

Are you self-hosted or cloud?

There is a related issue for this [here](https://github.com/Budibase/budibase/issues/11044).

If you are self-hosted you can follow these steps.

1. Find the relevant Redis container name by looking for redis-service or similar in your list of running containers
docker ps
2. Stop the Redis container
docker stop redis-service
3. Remove the Redis container
docker rm redis-service
4. Bring down all of the other containers and then start them up again - a fresh Redis container should be brought up with no pending user invites
docker compose down
docker compose up

This should be a lot easier hopefully when the Github issue linked early is sorted it should be as simple as revoking.

Currently the best **workaround** would be to use the `+email` trick if your email provider allows it. E.g. `email+test@email.com`. This will allow another email to be sent to the same email address. If you are worried about the same issue happening again. You can instead opt to using the generate password method of creating a user account. Once created simply send the generated password to the user, after entering they will be prompted to change it upon logging in.

I hope this all makes sense and is helpful.

**Reply by Freundschaft** [reply]
this doesnt seem to work anymore, I tried restarting the redis container but it didnt change anything
